doc/api/documentation.md

```diff
@@ -49,44 +45,47 @@ Backwards compatibility across major versions should not be expected.
 Stability: 1 - Experimental
 This feature is still under active development and subject to non-backwards
 compatible changes, or even removal, in any future version. Use of the feature
-is not recommended in production environments. Experimental features are not
-subject to the Node.js Semantic Versioning model.
+is not recommended in production environments. **Experimental features are not
+subject to the Node.js Semantic Versioning model.**
```

Comment:
I'm generally opposed to adding bold text except in extremely compelling situations. We waaaaaaay overuse typographical emphasis to the point that it has become mostly meaningless.

I'm especially opposed to it here because this is jargon a lot of people won't understand. I know you didn't write this text here, but "are not subject to the Node.js Semantic Versioning model" is vague and kind of misleading. There's nothing Node.js-specific about semantic versioning, and it's not clear that it could correctly be called a _model_ or what that means/implies.

This text is redundant anyway. The very first sentence says things plainly: "subject to non-backwards compatible changes, or even removal, in any future release." Maybe dispense with this last jargon-y sentence and (if we must emphasize something) emphasize that text instead?

Reply:
^^^^ non-blocking objection, by the way. Just comments.